tests/test_restful_api.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of ClaimStore.
+# Copyright (C) 2015 CERN.
+#
+# ClaimStore is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# ClaimStore is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ClaimStore; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307,
+# USA.
+
+"""claimstore.modules.claims.restful test suite."""
+
+import json
+import os
+
+import pytest
+
+from webtest import TestApp
+
```

Comment:
Pylon's `WebTest` looks nice, I did not know about it.  An alternative would be to use `Flask-Testing` e.g. as in [our Flask-IIIF test suite](https://github.com/inveniosoftware/flask-iiif/tree/master/tests), but we can definitely stay with `WebTest`.